src/plugin-styles.scss

```diff
@@ -22,4 +22,15 @@ $modal-footer-height: 70px;
 			object-fit: cover;
 		}
 	}
+
+	&__plugin-sidebar {
+		&__about-button {
+			color: #949494;
```

Comment:
Should we use variables from`@wordpress/base-styles` here instead of hard-coded values? Looks like this would be `$gray-600`.

Reply:
Done here: https://github.com/WordPress/create-block-theme/pull/667/commits/38c85125eb76b142a95c9081bbf7315f971eca4a